var/spack/repos/builtin/packages/py-pypulse/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPypulse(PythonPackage):
+    """Pulse is a package used to handle PSRFITS files and perform subsequent
+    analyses on pulse profiles."""
+
+    homepage = "https://github.com/mtlam/PyPulse"
+    pypi     = "PyPulse/PyPulse-0.0.1.tar.gz"
+
+    version('0.1.1', sha256='89cb362d2108215784f5b0df8e79311f7856f0da705ce4d77d4e884ca6c651b4')
+    version('0.0.1', sha256='239823737644bdf1e09e23e81b3fc439db096aa589581d9fa2b5717f1572e75b')
+
+    depends_on('python@2.7:,3:',         type=('build', 'run'))
```

Comment:
`2.7:,3:` is equivalent to `2.7:` (`2.7:,3.5:` is also equivalent to `2.7:`), but I don't see anything in the package that suggests that they support Python 2.

Reply:
https://github.com/mtlam/PyPulse/blob/master/README.md?plain=1#L15

Reply:
Ah, in that case you want:
```suggestion
    depends_on('python@2.7:',         type=('build', 'run'))
```